skrub/_reporting/_data/templates/dataframe-sample.css

```diff
@@ -3,23 +3,75 @@
 
 .table-with-selectable-cells {
     margin: 2px;
+    border: none;
+}
+
+.table-with-selectable-cells tbody {
+    border: 1px solid var(--mediumg);
+}
+
+.table-with-selectable-cells thead th {
+    vertical-align: top;
+    border: 1px solid var(--mediumg);
 }
 
 /* The column containing the selected cell has green stripes. */
 /* The rest of the table has gray stripes added by pure.css */
 
-tr:nth-child(2n -1) .table-cell[data-is-in-active-column] {
-    background-color: var(--mediumgreen);
+.table-cell[data-is-in-active-column] {
+    /* !important to override pure.css */
+    background-color: var(--active-column-color) !important;
```

Comment:
Can you give a precise css path to avoid the important?
Precise css selectors override imprecise ones